lab/skel_graph/astar/main.cpp

```diff
@@ -0,0 +1,216 @@
+// Copyright Radu Nichita 2020 radunichita99@gmail.com
+
+
+// Algoritmul A* - complexitate O(b ** d), b = branching factor, d = depth.
+
+#include <bits/stdc++.h>
+
+using Matrix = std::vector<std::vector<int>>;
+using Parents = std::vector<std::vector<std::pair<int, int>>>;
```

Comment:
I would remove these two aliases as you don't use them in multiple places and maybe the people is socked to see a type Matrix. I think we only need a Cell type as we reuse very often the pair<int, int> type.